content/java/concepts/hashset/hashset.md

```diff
@@ -0,0 +1,216 @@
+---
+Title: 'HashSet'
+Description: 'Used to store an unordered set of unique items of a given datatype.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Data Structures'
+  - 'Data Types'
+  - 'Collections'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+A `HashSet` is used to store an unordered set of unique items of a given [data type](https://www.codecademy.com/resources/docs/java/data-types).
+
+## Syntax
+
+`HashSet` is a member of the `java.util` package which must be imported first.
+
+```java
+import java.util.HashSet;
+
+HashSet<DataType> h = new HashSet<DataType>();
+```
+
+Where `h` is the newly created `HashSet` and `DataType` is the data type that will be stored in it.
```

Comment:
Can we include that `new` is a keyword? Doesn't have to be the same as the suggestion below.
```suggestion
Where `h` is a `HashSet` created with the `new` keyword and `DataType` is the data type that will be stored in it.
```